### ISSUE TITLE
ImportError: No module named 'xmltodict'

### ISSUE THREAD

**Issue by RPiAwesomeness** [Innectic/beam-streamer#1]
Yay first issue! Either way, when I install bstreamer via cloning the repo & doing `sudo pip3 install .`, and then trying to run it via
`➜  beam-streamer git:(master) bstream monstercat`

I get this traceback:

```python3
Traceback (most recent call last):
  File "/usr/local/bin/bstream", line 4, in <module>
    from beamstreamer import beam_streamer
  File "/usr/local/lib/python3.5/dist-packages/beamstreamer/beam_streamer.py", line 4, in <module>
    import xmltodict
ImportError: No module named 'xmltodict'
```

This is probably an easy fix, you should be able to add this module to one of the `setup.[py, cfg]` files and have it be installed by pip.

**Comment by Innectic** [comment]
Working on a fix now :D

**Comment by Innectic** [comment]
Fixed in release 1.2.1
